packages/shopping-cart/src/types.ts

```diff
@@ -614,6 +614,7 @@ export interface RequestCartProductExtra extends ResponseCartProductExtra {
 	import_dns_records?: boolean;
 	signup?: boolean;
 	headstart_theme?: string;
+	feature_slug?: string;
```

Comment:
Your comment above made me think that it might be a good idea to add a comment directly above this property in the type to explain its purpose. That way the next developer to see it will know what it's for without having to search for this PR. (Honestly we should do that for every property here but we've got to start somewhere.)